tests/app/helpers.py

```diff
@@ -79,36 +83,22 @@ def setup_authorization(app):
         HTTP_AUTHORIZATION='Bearer {}'.format(app.config['DM_SEARCH_API_AUTH_TOKENS']))
 
 
-def make_standard_service(**kwargs):
-    service = {
-        "id": "id",
-        "lot": "LoT",
-        "serviceName": "serviceName",
-        "serviceDescription": "serviceDescription",
-        "serviceBenefits": "serviceBenefits",
-        "serviceFeatures": "serviceFeatures",
-        "serviceCategories": ["serviceCategories"],
-        "supplierName": "Supplier Name",
-        "publicSectorNetworksTypes": ["PSN", "PNN"],
-    }
-
-    service.update(kwargs)
-
-    return {
-        "document": service
-    }
-
-
 def make_search_api_url(data, type_name='services'):
     # currently, data may contain either "document" or "service" (as a backward compatibility shim)
-    return '/index-to-create/{}/{}'.format(type_name, str(data.get('document', data.get('service'))['id']))
+    return '/test-index/{}/{}'.format(type_name, str(data.get('document', data.get('service'))['id']))
 
 
 def assert_response_status(response, expected_status):
+    __tracebackhide__ = True
     assert response.status_code == expected_status, "Expected {} response; got {}. {}".format(
         expected_status, response.status_code, response.get_data(as_text=True)
     )
 
 
-def get_json_from_response(response):
-    return json.loads(response.get_data())
+def create_services(number_of_services):
```

Comment:
If we consolidate the setup methods of `TestSearchResultsOrdering` and `TestSearchEndpoint` (they do the same thing) then `create_services` is only used in one place.
Happy to do as separate PR

Reply:
Interesting